src/myUtil.ml

```diff
@@ -104,6 +104,26 @@ module ResultMonad = struct
   let return v = Ok(v)
 
   let err e = Error(e)
+
+  let ( let+ ) = ( >>= )
```

Comment:
`(let*)` for `(>>=)` would be conventional.

https://v2.ocaml.org/manual/bindingops.html#ss%3Aletops-rationale

Reply:
Thanks! I will replace `let+` with that.